src/bentoml/_internal/configuration/containers.py

```diff
@@ -177,10 +177,16 @@ def bentoml_home() -> str:
         bentos = os.path.join(home, "bentos")
         models = os.path.join(home, "models")
         envs = os.path.join(home, "envs")
+        tmp_bentos = os.path.join(home, "tmp")
```

Comment:
Why do we need a persistent tmp bento store? seems a bit unintuitive 😄

Reply:
These Bentos are meant for a temporary cache and not suppose to be visible to users.